llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -411,6 +411,11 @@ class Value {
 
   Type *getType() const;
 
+  /// \Returns the expected type of \p V. For most Values this is equivalent
```

Comment:
the expected type "of \p V" 
Perhaps "the expected type of this Value" ?

Reply:
obviated by refactoring.